src/main/webapp/app/exercises/shared/exercise-hint/manage/exercise-hint.service.ts

```diff
@@ -49,9 +51,10 @@ export class ExerciseHintService implements IExerciseHintService {
 
     /**
      * Creates an exercise hint
+     * @param exerciseId of the exercise
      * @param exerciseHint Exercise hint to create
      */
-    create(exerciseHint: ExerciseHint): Observable<ExerciseHintResponse> {
+    create(exerciseId: number, exerciseHint: ExerciseHint): Observable<ExerciseHintResponse> {
         exerciseHint.exercise = ExerciseService.convertDateFromClient(exerciseHint.exercise!);
         exerciseHint.type = 'text';
         if (exerciseHint.exercise.categories) {
```

Comment:
Please use the passed exerciseId below instead of the one from the exercise hint